model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -53,7 +53,16 @@ def get_concat_after_split(split: Node) -> Optional[Node]:
 
     dest = split.out_port(0).get_destinations()[0].node
     # The transformation is applicable, only if next node is Concat.
-    return dest if dest.soft_get('type') == 'Concat' else None
+    if dest.soft_get('type') != 'Concat':
+        return
+
```

Comment:
Also, I think we need to make a comment for this function that it checks a sub-graph `Split->Concat` where Split just has one output and Concat has just one producer. Now it is not said explicitly.

Reply:
Done.